packages/cli/src/tools/config/__mocks__/resolveNodeModuleDir.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @flow
+ */
+
+const path = require('path');
+
+export default function resolveNodeModuleDir(
+  root: string,
+  packageName: string,
+): string {
+  return path.join(root, 'node_modules', packageName);
```

Comment:
Shouldn't this use `require.resolve` as per PR desc? Otherwise, it seems to be same as before the PR, unless I am blind at this time of the day!

Reply:
psst, this is a mock

Reply:
so I was right XD I am blind

Reply:
don't worry, I had the same wtf moment 😅